### DISCUSSION TITLE
Mi Wireframe y mi UI, que les parece estara bien, no tengo idea. Me apoye en consejos y algunas cosas que leí por allí

### DISCUSSION THREAD

**Discussion by ChanLuis** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#197] [Q&A] [answered]
![Wireframe UX](https://user-images.githubusercontent.com/99062113/156286047-43e9b057-3101-45b6-a42a-c233e3627f59.jpg)
![UI ](https://user-images.githubusercontent.com/99062113/156286106-b9fd64b6-983e-44f3-8099-2a003e2e314a.jpg)

**Comment by JuanSalvador73** [comment, marked as answer]
De hecho está bastante bien. Una vez que tienes el diagrama de modelo de tu App, ya tienes la lógica a seguir para desarrollar los pasos en UX y de ahí el diseño final en UI. 

Si haces un cambio de lógica, revisa el modelo original para ver como impactará en la secuencia de pasos y hacer los cambios o correcciones.

Buen trabajo.

**Reply by ChanLuis** [reply]
Gracias mano, es que la verdad estoy haciendo todo sin estar muy seguro. Sí del diagrama me base, ya con eso se facilita imaginar el funcionamiento. Gracias

**Reply by JuanSalvador73** [reply]
Excelente, que bueno que ha servido. Saludos :D

**Reply by Eholguin05** [reply]
en que aplicacion o pagina lo hiciste?

**Reply by MARYCHS** [reply]
Te quedo excelente, además nos sirvió para darnos idea a los que andamos perdidos.  gracias por compartir

**Comment by LuceroLuciano** [comment]
Te quedo super bien!  

Solo te faltaría mostrar el apartado del administrador donde pueda ver las demandas que tiene y los  ingresos en el dashboard como lo mencionaba el ejercicio, así como el envió de mensajes por parte del administrador(abogado) que le llegaran al usuario(cliente) por medio de correos. 

Saludos :)
